Plugins/FormatSwift/Plugin.swift

```diff
@@ -96,6 +115,32 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
 
 }
 
+#if canImport(XcodeProjectPlugin)
+extension AirbnbSwiftFormatPlugin: XcodeCommandPlugin {
+
+  func performCommand(context: XcodePluginContext, arguments: [String]) throws {
```

Comment:
Nice work @calda ! I see that in our current implementation we are only considering the arguments in the standard plugin implementation.

In Xcode 14 it's possible to pass arguments as well. Can we honor those arguments passed in from Xcode for completeness?

<img width="531" alt="Screen Shot 2022-07-28 at 6 03 11 PM" src="https://user-images.githubusercontent.com/1791049/181661565-a78b6c64-d34d-43a3-8e25-2790bb2cfcfa.png">

Reply:
I think it sort of makes sense for the SPM package plugin and Xcode plugin to have different sets of arguments, especially considering the supported functionality is different (e.g. there isn't a feasible implementation of `--paths` for the Xcode plugin). Also since you have to add the arguments manually each time you invoke the tool in Xcode, I somewhat assume excluding individual files wouldn't be a common use case. It wouldn't hurt to support it though.

Reply:
I see what you're saying @calda . I mostly want to make sure we're intentional.

> there isn't a feasible implementation of `--paths` for the Xcode plugin

Can you elaborate on why this isn't possible? Asking to learn.

Reply:
I was thinking of the difference between SPM targets and Xcode targets -- SPM targets correspond directly to folders, so have a `directory` path for these input paths to be resolved against. On the other hand Xcode targets are just a collection of files so don't have a `directory` path. The Xcode project itself does have a `directory`, though, now that I look at it, so you could resolve paths against that directory.

Reply:
Makes sense. I think it's reasonable to ignore `--files` when within Xcode.